src/ripple/ledger/impl/PaymentSandbox.cpp

```diff
@@ -186,15 +186,26 @@ PaymentSandbox::balanceHook (AccountID const& account,
     {
         auto delta = amount.zeroed ();
         auto lastBal = amount;
+        auto minBal = amount;
         for (auto curSB = this; curSB; curSB = curSB->ps_)
         {
             if (auto adj = curSB->tab_.adjustments (account, issuer, currency))
             {
                 delta += adj->debits;
                 lastBal = adj->origBalance;
+                if (lastBal < minBal)
+                    minBal = lastBal;
             }
         }
         adjustedAmt = std::min(amount, lastBal - delta);
+        if (amendmentRIPD1368(info().parentCloseTime))
```

Comment:
I rebased this on my "Add the config preset features to the view" PR and I'm putting it on a feature now.